zuul.d/edpm.yaml

```diff
@@ -35,6 +35,8 @@
       network_isolation: true
       crc_attach_default_interface: true
       bmo_setup: false
+      # This should be set here since is associated with parent job (base-crc) and not the scenario
+      cifmw_openshift_setup_kubeconfig: "/home/zuul-worker/.crc/machines/crc/kubeconfig"
```

Comment:
The variable we created for this purpose is cifmw_kubeconfig

Reply:
sorry, let me update that

Reply:
Don't worry, this fault went through the CI, I think we should enable the edpm job in case we touch this role too